src/main/java/org/folio/rest/impl/LoginAPI.java

```diff
@@ -307,27 +302,17 @@ private Future<String> handleLogin(LoginCredentials credentials, String tenantId
             .map(obj -> ((JsonObject)obj).mapTo(UserTenant.class))
             .collect(toList());
           if (userTenants.isEmpty()) {
-            // No matching tenant - if a tenant was specified, return an error
-            if (StringUtils.isNotBlank(credentials.getTenant())) {
-              logger.info(MISSING_USER_TENANT_ASSOCIATION_LOG, credentials.getUsername(), credentials.getUserId(),
-                  credentials.getTenant());
-              asyncResultHandler.handle(Future.succeededFuture(PostAuthnLoginResponse.respond422WithApplicationJson(
-                  LoginAPI.getErrors(MISSING_USER_TENANT_ASSOCIATION, CODE_MISSING_USER_TENANT_ASSOCIATION))));
-              promise.fail(new Exception(MISSING_USER_TENANT_ASSOCIATION));
-            } else {
-              promise.complete(tenantId);
-            }
+            promise.complete(tenantId);
           } else if (userTenants.size() > 1) {
```

Comment:
I remembered that we have one more issue with using system users, if for example we have 10 system users created in 10 different tenants, so table user_tenant in central tenant would contain 10 records with the same username but different tenantIds. So we will have issue multiple matching users in this case. 
To resolve it:

- we can add tenantId to LoginCredentials dto to specify exact tenant, but in this case we need to modify all places in Folio where system users using
- modify existing PR to get tenantId from headers in case of multiple matching users in user_tenant table found.
I suggest to go with second direction